melos.yaml

```diff
@@ -86,18 +86,18 @@ scripts:
   dev:designer:
     run: |
       melos exec -c 1 -- \
-        "flutter run -d chrome --web-port 3001 --dart-define=STUDYU_ENV=.env.selfhost"
+        "flutter run -d chrome --web-port 3001 --dart-define=STUDYU_ENV=.env.hpi"
     packageFilters:
       scope: studyu_designer_v2
-    description: Runs designer in Chrome on port 3001 with selfhost env for local development
+    description: Runs designer in Chrome on port 3001 with hpi env for local development
```

Comment:
I was using these workflows for development with my localhost Supabase instance before (i.e. to keep it the same, it should now be `.env.local`). Is `.env.hpi` the production environment? Is it safe to use this for local development?

Reply:
I'd recommend to use `.env.local` for now for local development as before. `.env` is currently the production environement and `.env.hpi` will be used for remote testing the current dev branch in the following weeks (but I might reset the db at any time so please do not store important stuff there). After migration, `.env.hpi` will replace the current `.env` environment, and `.env.dev` will be used for remote testing. That's the plan at least. :D

Reply:
Cool! Then let's switch the env for `dev:designer|app` to `.env.local`